File: application/backend-credit-card-enrollment/backend-java/src/main/java/cloud/ambar/creditcard/enrollment/commandhandler/RequestEnrollmentCommand.java
package cloud.ambar.creditcard.enrollment.commandhandler;

import cloud.ambar.common.commandhandler.Command;
import lombok.Builder;
import lombok.Getter;
import lombok.NonNull;

@Builder
@Getter
public class RequestEnrollmentCommand extends Command {
    @NonNull private String userId;
    @NonNull private String productId;
    @NonNull private Integer annualIncomeInCents;
}

File: application/backend-credit-card-enrollment/backend-c#/CreditCard/Enrollment/Command/RequestEnrollmentCommand.cs
namespace CreditCardEnrollment.CreditCard.Enrollment.Command;

public class RequestEnrollmentCommand : Common.Command.Command {
    public required string SessionToken { get; init; }
    public required string ProductId { get; init; }
    public required int AnnualIncomeInCents { get; init; }
}